app/conf/CatalogService.scala

```diff
@@ -0,0 +1,21 @@
+package conf
+
+import com.gu.config.SubsV2ProductIds
+import com.gu.memsub.subsv2
+import com.gu.okhttp.RequestRunners.futureRunner
+import com.gu.zuora.{ZuoraApiConfig, rest}
+import com.typesafe.config.Config
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scalaz.std.scalaFuture._
+
+object CatalogService {
+  def fromConf(config: Config, env: String)={
```

Comment:
Small style thing - best not to use abbreviations - so consider renaming to fromConfig 

* IMO - Config is an allowed abbreviation of Configuration just like Id is an allowed abbreviation of Identifier.